snap/snapcraft.yaml

```diff
@@ -640,6 +652,11 @@ parts:
 
       done
 
+      # For securing Kong's admin APIs, we also need to install kong-admin-config's template yml
+      # into secretstore-setup service
```

Comment:
Comment doesn't fully explain what is going on.

Reply:
not sure what you want me to add here: it is just try to copy that template yml to the proper location.

Reply:
It doesn't say WHY secretstore-setup needs this template.

Reply:
got you, I'll add that, thanks.

Reply:
added